packages/core/src/metrics/platform/CordovaAppMetrics.ts

```diff
@@ -1,15 +1,16 @@
 import { AppMetrics, Metrics } from "../model";
 
+// plugin's JS interface
+declare var aerogear: {
```

Comment:
aerogear seems unfortunate as we will not be able to declare more on this.

Reply:
What do you mean we won't be able to declare other functions? Anyway I think this declaration is no longer needed, since we can remove the JS wrapper -> (https://github.com/aerogear/aerogear-js-sdk/pull/66#discussion_r184704632)

Reply:
When having 2 declarations with the same name previous declaration will be shadowed causing some hard to find compilation errors. Been there.. :)